seeds/clipper/arbitrum/clipper_arbitrum_trades_test_data.csv

```diff
@@ -0,0 +1,6 @@
+blockchain,project,version,block_date,block_time,token_bought_symbol,token_sold_symbol,token_pair,token_bought_amount,token_sold_amount,token_bought_amount_raw,token_sold_amount_raw,amount_usd,token_bought_address,token_sold_address,taker,maker,project_contract_address,tx_hash,tx_from,tx_to
+arbitrum,clipper,1,2022-12-15T00:00:00Z,2022-12-15T22:05:01Z,USDC,USDT,USDC-USDT,5.490002,5.49,5490002,5490000,5.491528220556,0xff970a61a04b1ca14834a43f5de4533ebddb5cc8,0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,0x52fb93047acf981c64a3d54f9fb63d1634293766,,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c,0x4e8b282691842b546869ff2ffb6e33e515d609bced31527bc049f92e6e00c831,0x52fb93047acf981c64a3d54f9fb63d1634293766,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c
+arbitrum,clipper,1,2022-12-15T00:00:00Z,2022-12-15T23:24:37Z,USDT,WETH,USDT-WETH,10.474675,0.0083,10474675,8300000000000000,10.485767680824999,0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0x90fa8de4f5f0058d3a8387903ed66d7b74072060,,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c,0x3c63866b0a50ef829cf1431b90116fad82e44823228dfa57a6fb43eda552304b,0x90fa8de4f5f0058d3a8387903ed66d7b74072060,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c
+arbitrum,clipper,1,2023-02-15T00:00:00Z,2023-02-15T11:00:35Z,DAI,USDT,DAI-USDT,114.45187511833501,114.56,114451875118335020000,114560000,114.46961515897834,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,0xd358725d4fa42869acc1bc49e6326adb22828e29,,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c,0x52f3a5f2a8217e8bba343841bd379c076cfa06c1279d64ee43f891fc4aaedebc,0xd358725d4fa42869acc1bc49e6326adb22828e29,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c
```

Comment:
Maybe this seed is simply too large to parse...

Reply:
Strangely, I see only 3 lines, although the test contains 5:

select * from test_schema.git_b77df63d_clipper_arbitrum_test

Also, only the `token_bought_amount_raw_test` is false.
Please advise what I can do?

Reply:
@Hosuke I removed 2 lines, now it has 3 (instead of 5)

Reply:
I will rerun the test and see what happen then

Reply:
@Hosuke I see the following in the log:
Error: The operation was canceled.
should I do anything from my side?

Reply:
I have reran the test:
https://github.com/duneanalytics/spellbook/actions/runs/4324749336/jobs/7563350911#step:10:319

Reply:
Something strange is with the `token_bought_amount_raw`. It is now declared as bigint, and also, I've removed 2 lined from the seed data. In the test result there are still 5 lines and the value of `114451875118335020000` represented as `9223372036854776000`:
```
select 
tx_hash,
model.token_bought_amount_raw,
seed.token_bought_amount_raw as seed_raw
from test_schema.git_4a9af893_clipper_arbitrum_trades_test_data seed
inner join test_schema.git_4a9af893_clipper_v1_arbitrum_trades model using (tx_hash)
```

Could it be the old branch?

Reply:
@Hosuke please advise

Reply:
Done

Reply:
We may either parse `token_bought_amount_raw` as string and try again or remove the tests/clipper/arbitrum/clipper_arbitrum_test.sql. The current model is actually mergeable, the problem is due to seed and custom testing.

Reply:
what is the right way to  parse token_bought_amount_raw as string?
or should I remove the tests?

Reply:
If you remove the test, we may make the PR `ready-for-merging`. Or we may parse the seed `token_bought_amount_raw` column as `string`, and apply cast(* as decimal(38,0)) to both values during comparison in the test.(not sure if the test will pass in this way)